packages/core/src/CloseButton/__snapshots__/CloseButton.spec.js.snap

```diff
@@ -55,7 +62,9 @@ exports[`CloseButton renders without props 1`] = `
 }
 
 <button
+  aria-label="close"
```

Comment:
😱

Reply:
Yeah, I have a couple issues to iron out haha

Reply:
this is good. it's bad that it was previously getting suppressed on derivative buttons.